tests/janitor/test_mozilla_plugin.py

```diff
@@ -0,0 +1,24 @@
+import os
+import unittest
+
+import mock
+from ubuntucleaner.janitor.mozilla_plugin import FirefoxCachePlugin
+
+
+class TestMozillaCachePlugin(unittest.TestCase):
+    def setUp(self):
+        self.firefox_plugin = FirefoxCachePlugin()
+
+    def test_firefox_plugin(self):
+        self.assertTrue(
+            os.path.expanduser('~/.mozilla/firefox/5tzbwjwa.default'),
+            self.firefox_plugin.get_path()
+        )
+
+    @mock.patch('ubuntucleaner.settings.common.RawConfigSetting.sections',
+                mock.Mock(side_effect=Exception))
+    def test_firefox_plugin_path_exception(self):
```

Comment:
I'm not sure I understand this test, we're patching to raise an exception as a side effect and then asserting on `get_path()` :thinking:

Reply:
well I'm testing get_patch with the excenario of the try..except that's why I force the RawConfigSetting.sections to raise an exception.
In the end this case ends up returning a valid path anyway.